server/deckUtils.go

```diff
@@ -88,3 +88,12 @@ func generateShuffledDeck(numPlayers int) []model.Card {
 
 	return shuffleCards(deck)
 }
+
+// Returns true if a card is a number card
+func isNumberCard(card model.Card) bool {
+	if card.Value == "W" || card.Value == "W4" || card.Value == "S" || card.Value == "D2" || card.Value == "R" {
```

Comment:
I wonder if a switch would be easier to read here:
```
switch (card.Value){
  case "W":
  case "W4":
  case "S":
  case "D2":
  case "R":
    return false
    break
  default
    return true
}
```

Reply:
Golang does not have fallthrough unless you specifically add the `fallthrough` keyword. I think by the time I add all the `fallthrough`s it might not be as readable as the `if` condition. What are your thoughts?

Reply:
Totally missed that this was on the `go` side! You are absolutely correct.